code/modules/admin/admin_verbs.dm

```diff
@@ -1166,6 +1168,20 @@ var/list/admin_verbs_cciaa = list(
 	log_and_message_admins("admin-wiped [key_name_admin(target)]'s core.")
 	target.do_wipe_core()
 
+/client/proc/end_round()
+	set name = "zEnd Roundz"
```

Comment:
zEnd Roundz ??

Reply:
puts it at the bottom of the list in the top right

Reply:
Yeah don't do that, just do End Round please